agb/src/display/affine.rs

```diff
@@ -173,17 +208,58 @@ impl From<AffineMatrixBackground> for AffineMatrix {
     }
 }
 
-impl Default for AffineMatrix {
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[repr(C, packed(4))]
+/// An affine matrix that can be used in affine objects
+pub struct AffineMatrixObject {
+    // Internally these can be thought of as Num<i16, 8>
+    a: i16,
+    b: i16,
+    c: i16,
+    d: i16,
```

Comment:
you should be able to actually make these `Num<i16, 8>` given the representation